kclvm/sema/src/builtin/decorator.rs

```diff
@@ -21,6 +21,7 @@ macro_rules! register_decorator {
 
 register_decorator! {
     deprecated => Type::function(
+        Some("Decorator".to_string()),
```

Comment:
Considering to use a `FuncKind` enum or constants.

Besides, I think it is a display type, not a display package there.